engine/src/conversion/analysis/fun/mod.rs

```diff
@@ -198,25 +199,40 @@ impl Default for ReturnTypeAnalysis {
     }
 }
 
+pub(crate) struct FnStructAnalysis {
```

Comment:
Please could you rename to `PodAndConstructorAnalysis`. I know that's wordy, but I want to make a clear distinction between analysis results of _types_ vs _functions_ and this sounds a bit like an analysis of a function right now.

Reply:
Done